packages/amplication-client/src/VersionControl/PendingChange.tsx

```diff
@@ -80,6 +80,13 @@ export const changeOriginMap = {
         type: "Service Settings",
         pluralTypeName: "Service Settings",
       },
+      [models.EnumBlockType.Package]: {
+        relativeUrl: `settings/packages`,
```

Comment:
Shouldn't this be `packages/${change.id}` ?

Reply:
@mulygottlieb, let's handle this in the next PR for UI. Does it sound ok?